docs/apis/avalanchego/avalanchego-release-notes.md

```diff
@@ -6,6 +6,18 @@
 
 :::
 
+## V1.9.5 ([View on GitHub](https://github.com/ava-labs/avalanchego/releases/tag/v1.9.6))
+
```

Comment:
We should add a description for this release as in 
<img width="1164" alt="Screen Shot 2022-12-19 at 9 12 55 AM" src="https://user-images.githubusercontent.com/12171681/208470329-5e97e878-c53a-4ab8-9cd3-34f7f94e41ac.png">

We should add it to 1.9.4 too.